### PR TITLE
Add animation to appbar panel titles

### LINKED ISSUE #1846: [FEATURE] Add animations to the map (how elements load)
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Who request this Feature

_No response_

### User Story

We need to add animations to how elements appear.

**Animate AppBar** 
- [x] Animate how Geolocator app, make the search appear from left.
- [x] Make the animation appear from the bell.
- [ ] About GeoView modal should appear from the icon. When you close it should close to the icon.

### Implementation UI

_No response_

### PR DESCRIPTION
**Pull request by cphelefu**
# Description

This PR adds fade animation to panel title in the appBar and also to the content.

Fixes #1846

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

https://cphelefu.github.io/geoview/demos-navigator.html?config=./configs/navigator/07-basic-appbar.json

# Checklist:

- [x] I have build __(rush build)__ and deploy __(rush host)__ my PR
- [x] I have connected the issues(s) to this PR
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have created new issue(s) related to the outcome of this PR is needed
-  ~~I have made corresponding changes to the documentation~~
-  ~~I have added tests that prove my fix is effective or that my feature works~~
-  ~~New and existing unit tests pass locally with my changes~~

- - -
This change is [Reviewable](https://reviewable.io/reviews/Canadian-Geospatial-Platform/geoview/2400)

### CODE REVIEW

File: packages/geoview-core/src/core/components/notifications/notifications-style.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const getSxClasses = (theme: Theme): any => ({
     maxHeight: '500px',
     overflowY: 'hidden',
     gap: '8px',
-    marginLeft: '15px',
+    marginLeft: '18px',
```
Comment (cphelefu) [inline packages/geoview-core/src/core/components/notifications/notifications-style.ts:13]: This just fixes the spacing of the notification panel

**Review by kaminderpal — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/Canadian-Geospatial-Platform/geoview/2400#-:-O2_Pat9-09TGWyskncH:bnfp4nl)* status: 0 of 2 files reviewed, 1 unresolved discussion (waiting on @jolevesq)

**Review by jolevesq — APPROVED** [review]
> Reviewed 1 of 2 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/Canadian-Geospatial-Platform/geoview/2400)* status: all files reviewed, 1 unresolved discussion (waiting on @jolevesq)

**Review by jolevesq — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/Canadian-Geospatial-Platform/geoview/2400#-:-O2_Ph184g3RnxYu4Rne:bnfp4nl)* status: all files reviewed, 1 unresolved discussion

**Pull request merged**